src/ucs/arch/x86_64/cpu.c

```diff
@@ -12,11 +12,15 @@
 #include <ucs/sys/sys.h>
 
 #define X86_CPUID_GET_MODEL       0x00000001u
+#define X86_CPUID_GET_BASE_VALUE  0x00000000u
+#define X86_CPUID_GET_EXTD_VALUE  0x00000007u
 #define X86_CPUID_GET_MAX_VALUE   0x80000000u
 #define X86_CPUID_INVARIANT_TSC   0x80000007u
 
 
 static inline void ucs_x86_cpuid(uint32_t level, uint32_t *a, uint32_t *b,
+                                 uint32_t *c, uint32_t *d) UCS_F_NOOPTIMIZE;
+static inline void ucs_x86_cpuid(uint32_t level, uint32_t *a, uint32_t *b,
```

Comment:
gcc warnings about this (says to set attribute for declaration in case function is inlined)